main.cpp

```diff
@@ -189,6 +220,22 @@ void Bicycle::stopBicycle()
     std::cout << "Bicycle is being stopped!\n";
 }
 
+void Bicycle::arbitraryFunc(int valueA, int valueB, int doABikeThing)
+{
+    int valueC = 10;
+    int valueD = 5;
+    int arbitraryMember = valueC + valueD;
+    int totalMembers = arbitraryMember * valueC + valueD + valueA + valueB;
+
+    while(doABikeThing < totalMembers)
+    {
+        ++doABikeThing;
+        std::cout << "Say something about bikes" << std::endl;
+        if(doABikeThing >= 5)
+        break;
```

Comment:
Indent the body of any `if` blocks, even if they're a single statement/single line
```
if( condition )
    doSomething();
```
not:
```
if( condition )
doSomething();
```
- [ ] revise